ui/src/importer/importerTaskDetails/ImportedDocuments.js

```diff
@@ -65,11 +72,47 @@ export class ImportedDocuments extends React.Component {
           isLoading: true,
         });
       }
+      this.calculateStatistics(response.data);
     } else {
       this.intervalId && clearInterval(this.intervalId);
     }
   };
 
+  calculateStatistics = data => {
+    const importStatistics = [];
+    importStatistics.push({
+      text: 'Mode',
+      value: <div className="ui label">{data.mode}</div>,
+    });
+    importStatistics.push({
+      text: 'Records',
+      value: data.loaded_entries + '/' + data.entries_count,
+    });
+    importStatistics.push({
+      text: 'Records created',
+      value: data.records.filter(record => record.action === 'create').length,
```

Comment:
is there any more efficient way than filtering all the records each time the data is refreshed ? Did you try on a file with 500-1000+ records, if the UI will be performing well and fast ?

Reply:
I will address this issue in a separated PR, as it involves important code changes and is quite bound to the filter process.

Reply:
The work is being done in this PR.
https://github.com/CERNDocumentServer/cds-ils/pull/603